modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/css/_tooltip.scss

```diff
@@ -15,4 +15,4 @@
 			right: calc(50% - #{$arrowOffset});
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Can you remove changes on this file?

Reply:
@interaminense just remove `}` in line 18?

Reply:
Can't you only discard changes?